bot/validations.ts

```diff
@@ -345,15 +340,13 @@ const validateInvoice = async (ctx: MainContext, lnInvoice: string) => {
     const latestDate = new Date(
       Date.now() + Number(process.env.INVOICE_EXPIRATION_WINDOW)
     );
-    if (!("MAIN_PAYMENT_AMT" in process.env)) throw Error("MIN_PAYMENT_AMT not found, please check .env file");
+    if (!("MIN_PAYMENT_AMT" in process.env)) throw Error("MIN_PAYMENT_AMT not found, please check .env file");
```

Comment:
@danfercf1 this looks like an unrelated change that should go in a separate PR?

Reply:
Yes, you are right the problem is that the test related to this implementation will not work. I've reported that problem in this issue [https://github.com/lnp2pBot/bot/issues/594](Issue #596)